src/components/LoanCards/LoanCardImage.vue

```diff
@@ -0,0 +1,63 @@
+<!--
+	issues:
+
+	* tracking click on image - do here or bubble up?
+		kvtrackevent="Lending|click-Read more|Photo|{{id}}|true"
+
+	* do we need to deal with image editing?
+		{{#is_admin}}
+			<div class="image-edit-buttons">
+				<a class="edit" href="{{image.edit_url}}" target="_blank"><svg class="icon icon-edit"><use xlink:href="#icon-edit"/></svg></a>
+				<a class="refresh" href="#"><svg class="icon icon-refresh"><use xlink:href="#icon-refresh"/></svg></a>
+			</div>
+		{{/is_admin}}
+ -->
+
+
+
+<template>
+	<a href="{{borrowerPageUrl}}">
+		<!-- current implementation has image as background to <a> rather than <img />
+	 			any reason to continue that?
+				also - do we need an alt tag, if so, what should be in it? "photo of {borrower-name}"-->
```

Comment:
I think it was a background image to help with image scaling and to have an appropriately sized container while the image is loading. That shouldn't be required for those things though.

+1 to that alt value